HelpSource/Classes/HenonC.schelp

```diff
@@ -43,17 +43,13 @@ Second value of x
 examples::
 code::
 // default initial params
-~defaultParms={ HenonC.ar(MouseX.kr(20, SampleRate.ir)) * 0.2 !2};
+{ HenonC.ar(MouseX.kr(20, SampleRate.ir)) * 0.2 !2}.scope(s);
 
 // mouse-control of params
-~mousControl={ HenonC.ar(SampleRate.ir/4, MouseX.kr(1,1.4), MouseY.kr(0,0.3)) * 0.2 !2};
+{ HenonC.ar(SampleRate.ir/4, MouseX.kr(1,1.4), MouseY.kr(0,0.3)) * 0.2 !2}.scope(s);
 
 // randomly modulate params
-~randModPars=({ HenonC.ar(SampleRate.ir/8, LFNoise2.kr(1, 0.2, 1.2), LFNoise2.kr(1, 0.15, 0.15)) ! 2 * 0.02 });
+({ HenonC.ar(SampleRate.ir/8, LFNoise2.kr(1, 0.2, 1.2), LFNoise2.kr(1, 0.15, 0.15)) ! 2 * 0.02 }).scope(s);
```

Comment:
no need for the round parentheses, you don't have them in the above.
also better to leave out the `s`, it is redundant.

`{ HenonC.ar(SampleRate.ir/8, LFNoise2.kr(1, 0.2, 1.2), LFNoise2.kr(1, 0.15, 0.15)) ! 2 * 0.02 }.scope`